.circleci/codeReview.yml

```
@@ -0,0 +1,111 @@
+          context: 
+            - circleci-agents 
+
+jobs:
+  pr-review-with-tracking:
+    docker:
+      - image: cimg/python:3.11-node
+    resource_class: medium
+    steps:
+      - checkout
+      
+      # Install Claude Code CLI 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)